icu4c/source/i18n/messageformat2_data_model.cpp

```diff
@@ -1091,13 +1091,13 @@ MFDataModel::MFDataModel(const MFDataModel::Builder& builder, UErrorCode& errorC
         U_ASSERT(builder.selectors != nullptr);
         int32_t numVariants = builder.variants->size();
         int32_t numSelectors = builder.selectors->size();
-        Variant* variants = copyVectorToArray<Variant>(*builder.variants, errorCode);
-        Expression* selectors = copyVectorToArray<Expression>(*builder.selectors, errorCode);
+        LocalArray<Variant> variants(copyVectorToArray<Variant>(*builder.variants, errorCode), errorCode);
+        LocalArray<Expression> selectors(copyVectorToArray<Expression>(*builder.selectors, errorCode), errorCode);
```

Comment:
Do these arrays own their contents?  If so, you need to account for that.

Reply:
I did.   I call orphan() to release the ownership

Reply:
orphan() releases ownership of the arrays themselves.  I was asking about the _elements_ of the arrays.  I don't know what `Variant` and `Expression` are.  But if I'm reading this right, they're inline structs, not pointers, and so there isn't an ownership issue here and I asked the question in error.  Sorry about that.